python/grass/jupyter/display.py

```diff
@@ -0,0 +1,101 @@
+
+# MODULE:    grass.jupyter.display
+#
+# AUTHOR(S): Caitlin Haedrich <caitlin DOT haedrich AT gmail>
+#
+# PURPOSE:   This module contains functions for non-interactive display
+#            in Jupyter Notebooks
+#
+# COPYRIGHT: (C) 2021 Caitlin Haedrich, and by the GRASS Development Team
+#
+#           This program is free software under the GNU General Public
+#           License (>=v2). Read the file COPYING that comes with GRASS
+#           for details.
+
+import os
+import folium
+import grass.script as gs
+
+class grassRenderer:
```

Comment:
Classes should be CamelCase, just run Pylint to get all these complaints sooner :-)